content/en/blog/2022/dotnet-instrumentation-metrics.md

```diff
@@ -0,0 +1,33 @@
+---
+title: OpenTelemetry .NET Automatic Instrumentation metric signal support
+linkTitle: .NET Auto-instrumentation Metrics
+date: 2022-07-07
+author: OpenTelemetry .NET Automatic Instrumentation Team
+---
+
+We're excited to announce the
+[0.2.0-beta.1 release](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/tag/v0.2.0-beta.1)
+of the
+[OpenTelemetry .NET Automatic Instrumentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation)
+which adds metric signal support!
+
+Now you can easily export metrics from:
+
+- .NET Runtime,
+- ASP.NET Core,
+- ASP.NET Framework,
+- HTTP clients (`System.Net.Http.HttpClient` and `System.Net.HttpWebRequest`),
+- measurements created using [`System.Diagnostics.Metrics`](https://docs.microsoft.com/en-us/dotnet/core/diagnostics/metrics-instrumentation).
+
+Over the next few months we plan to support:
+
+- support additional
```

Comment:
remove `support` since it's in the sentence above already ("we plan to support")